assets/dev/scss/editor/_control-scrubbing.scss

```
@@ -0,0 +1,27 @@
+.e-scrubbing {
+	
+	cursor: e-resize;
+
+	* {
+	   cursor: e-resize;
+	}
+
+	input.e-scrubbing-element, 
+	input.e-scrubbing-element:focus {
+		border-color: #{ darken($editor-info, 20%) } !important;
+	}
```

```suggestion
	input.e-scrubbing-element {
		&, &:focus {
			border-color: #{ darken( $editor-info, 20% ) } !important;
		}
	}
```

Try to remove the important (maybe use more specific selector)

---

Done.